client/src/App.js

```diff
@@ -1,8 +1,9 @@
 import React, { Component } from 'react';
 
 import SavedList from './Movies/SavedList';
-import MovieList from './Movies/MovieList';
+import MovieList from "./Movies/MovieList";
 import Movie from './Movies/Movie';
+import { Route } from "reacter-router-dom";
```

Comment:
Be careful with the name of the package you're importing. You can always check the package.json file to see which packages you have imported